src/csharp/Intel/Generator/Assembler/CSharp/CSharpAssemblerSyntaxGenerator.cs

```diff
@@ -116,6 +112,11 @@ sealed class CSharpAssemblerSyntaxGenerator : AssemblerSyntaxGenerator {
 								renderArgs.Add(new RenderArg(argName, argType, argKind));
 							}
 
+							// Use the rm register if no regArg was found
```

Comment:
This commit changed the code a little bit. I think we shouldn't require __byte_ptr unless it's required, eg. `inc [mem]` is ambiguous and we must use eg. `inc dword ptr [mem]` but for some instructions, there's only one Code value and the check isn't needed.

Eg. this code shouldn't require byte ptr:

```diff
		public void clflushopt(AssemblerMemoryOperand dst) {
			Code op;
-			op = Code.Clflushopt_m8;
+			if (dst.Size == MemoryOperandSize.BytePtr) {
+				op = Code.Clflushopt_m8;
+			} else {
+				throw NoOpCodeFoundFor(Mnemonic.Clflushopt, dst);
+			}
			AddInstruction(Instruction.Create(op, dst));
		}
```

Reply:
Yeah, I saw that, I will fix it.